packages/odyssey-react/src/components/Table/TableHeaderCell.tsx

```diff
@@ -34,7 +35,8 @@ const TableHeaderCell = React.forwardRef<Ref, Props>((props, ref) => {
   } = props;
 
   const componentClass = useCx(
-    format && `is-${tableClass}-${format}`,
+    props.scope === 'col' ? styles.headerCellCol : styles.headerCell,
+    format && styles[format],
```

Comment:
is there no sensible default for `format` similar to how we've handled `direction` below in `TableSortButton.tsx` ?

Reply:
The default currently has no opinion on styling. I'm tempted to refactor `format` to 'textFormat' because that's really what's happening in the two cases (numbers are aligned right, dates are forced to a single line) and it ends up being less ambiguous

Reply:
makes sense thank you